plugin-java/che-plugin-java-ext-jdt/che-jdt-ext-machine/src/main/java/org/eclipse/che/jdt/rest/JavaClasspathService.java

```diff
@@ -65,28 +65,42 @@ public boolean update(@QueryParam("projectpath") final String projectPath) throw
         return success;
     }
 
-    private boolean generateClasspath(final String projectPath, File dir) {
+    private boolean dependencyUpdateProcessor(final String projectPath, File dir) {
+        String command = MavenUtils.getMavenExecCommand();
+
+        ProcessBuilder classPathProcessBuilder = new ProcessBuilder().command(command,
+                                                                              "dependency:build-classpath",
+                                                                              "-Dmdep.outputFile=.codenvy/classpath.maven")
+                                                                     .directory(dir)
```

Comment:
This is a folder which contains configuration files (classpath, modules). 
.codenvy will be renamed as part of issue https://jira.codenvycorp.com/browse/IDEX-3480.